answerking_app/tests/test_categories.py

```diff
@@ -344,6 +328,76 @@ def test_put_invalid_details_returns_bad_request(self):
         self.assertEqual(expected_failure_error, actual)
         self.assertEqual(response.status_code, 400)
 
+    def test_put_invalid_item_returns_bad_request(self):
+        # Arrange
+        item: ItemType = {
+            "id": self.test_item_3.id + 500,
```

Comment:
maybe put -1 as the ID as we know it will never exist

Reply:
test and see. I ran it on develop branch in postman and got "detail": "Not found error". If it works I would change it.

Reply:
It works because there is nothing in the serializer that specifies that the ID should be positive, just that it should be an integer field. And because the ID is auto generated by Django models there will never be a negative or 0 integer ID. So should be fine.